script.cinemavision/lib/main.py

```diff
@@ -448,40 +526,43 @@ def fillOptions(self):
         for i in cinemavision.sequence.ITEM_TYPES:
             item = kodigui.ManagedListItem(
                 '{0}: {1}'.format(T(32530, 'Add'), i[1]),
-                thumbnailImage='small/script.cinemavision-{0}.png'.format(i[2]),
+                thumbnailImage='{0}small/script.cinemavision-{1}.png'.format(cvutil.THEME['theme.path'], i[2]),
                 data_source=i[0]
             )
-            item.setProperty('thumb.focus', 'small/script.cinemavision-{0}_selected.png'.format(i[2]))
+            item.setProperty('thumb.focus', '{0}small/script.cinemavision-{1}_Selected.png'.format(cvutil.THEME['theme.path'], i[2]))
+            item.setProperty('thumb.fill', '{0}small/script.cinemavision-{1}_Fill.png'.format(cvutil.THEME['theme.path'], i[2]))
             self.addItemControl.addItem(item)
 
-        item = kodigui.ManagedListItem(T(32531, 'Edit'), T(32531, 'Edit'), thumbnailImage='small/script.cinemavision-edit.png', data_source='edit')
-        item.setProperty('alt.thumb', 'small/script.cinemavision-edit.png')
-        item.setProperty('thumb.focus', 'small/script.cinemavision-A_selected.png')
+        basePath = cvutil.THEME['theme.path'] + 'options/script.cinemavision-'
+
+        item = kodigui.ManagedListItem(T(32531, 'Edit'), T(32531, 'Edit'), thumbnailImage=basePath + 'ModuleEdit.png', data_source='edit')
```

Comment:
if thumbnailImage is later used as the ListItem kwarg please drop and use only setArt

Reply:
Done in the `kodigui` lib, so this these and everywhere else will ultimately use setArt.